src/RelayServer.ts

```diff
@@ -927,9 +934,9 @@ latestBlock timestamp   | ${latestBlock.timestamp}
   async getAllHubEventsSinceLastScan(): Promise<Array<TypedEvent>> {
     const options = {
       fromBlock: this._lastScannedBlock + 1,
-      toBlock: 'latest',
+      toBlock: 'latest' as LatestTag,
```

Comment:
```suggestion
      toBlock: 'latest',
```

Reply:
Without this "cast", we get ts errors 🤷🏽‍♂️